src/data/relics.js

```diff
@@ -2,74 +2,79 @@ import images from '../assets/images';
 
 export const NONE_RELIC_ID = -1;
 
-export const RELIC_UNLOCK_THRESHOLDS = [0, 500, 1200, 2000, 4000, 7500, 15000, 24000];
+export const RELIC_UNLOCK_THRESHOLDS = [0, 750, 1200, 2000, 4000, 7500, 15000, 24000];
 
 export const PASSIVE_RELICS = [
   {
     exp: {
       multiplier: 5,
       increased: true,
+      isCombatDoubled: false,
```

Comment:
should we use a multiplier value instead of a flag so it's easier to tweak later if they change the scaling? `combatMultiplier: 1` and so on